bijlage03_Versienummering.md

```diff
@@ -0,0 +1,81 @@
+
+## Versioneringsmethodiek
+Per document wordt met `[documentnaam] X.Y.Z` de versie aangegeven.
+Met `X.Y.Z` wordt gerefereerd aan major (X) en minor (Y) releases en (Z) patches,
+dit wordt hieronder toegelicht.
+
+* PATCH wordt verhoogd bij correcties.
+* MINOR wordt verhoogd bij wijzigingen waarbij uitwerkingen (implementaties) volgens de vorige versie van de standaard ook voldoen aan de normen/eisen van de nieuwe versie van de standaard.
+* MAJOR wordt verhoogd als de nieuwe versie van de standaard zodanig wijzigt dat uitwerkingen (implementaties) volgens de vorige versie van de standaard niet meer voldoen aan de normen/eisen van de nieuwe versie van de standaard.
+
+### Patch Releases
+In een patchrelease worden wijzigingen doorgevoerd die de technische
+specificatie niet raken. Dit kunnen tekstuele wijzigingen zijn of
+inhoudelijke indelingen van de documenten. De wijzigingen worden
+vastgelegd in release notes. Een patch releases wordt door de beheerder
+op eigen initiatief of op aanwijzingen van gebruikers doorgevoerd en gepubliceerd.
+Een patchrelease wordt aan het Technisch Overleg ter kennisgeving medegedeeld.
+Een nieuwe patchrelease vervangt een eerdere versie in zijn geheel.
+
+### Minor releases
+Een minor release geeft aan dat de nieuwe versie van de standaard zodanig is gewijzigd dat een implementatie van de voorgaande versie van de standaard voldoet aan de regels van de nieuwe versie. In een minor release kunnen wijzigingen doorgevoerd worden die de technische
+specificatie van een koppelvlak raken (bijvoorbeeld nieuwe functionaliteit). 
+Voor Minor Releases wordt een uitgebreid vaststellingsprocedure gevolgd
+(conform het beheermodel van de standaard) en er kan in overleg met de deelnemers
+van het Technisch Overleg tot een migratiepad worden besloten. Dit migratiepad
+wordt in de release meegenomen.
+
+### Major Releases
+Een major release geeft aan dat de nieuwe versie van de standaard zodanig is gewijzigd dat een implementatie van de voorgaande versie van de standaard **niet** voldoet aan de regels van de nieuwe versie.
+Bijvoorbeeld de overgang naar nieuwe externe
+(meestal internationale) standaarden binnen een bestaand profiel. Als hierbij het
+functionele toepassingsgebied van de standaard, waarvoor het pas-toe-of-leg-uit
+regime geldt, verandert, dan wordt eerst de uitgebreide vaststellingsprocedure
+gevolgd en vervolgens de procedure van het Forum Standaardisatie.
+
+<aside class="example" title="Grote wijzigingen voor Digikoppeling">
+Bij de overgang naar een andere externe standaard binnen een bestaand
+profiel kan men denken aan een overgang naar HTTP 2.0 of SOAP 1.2 binnen
+Digikoppeling WUS koppelvlakspecificatie.
+Het toevoegen van een geheel nieuw profiel kan voor Digikoppeling kan bestaan
```

Comment:
De zinsnede "of de toevoeging van een geheel nieuw profiel" is uit het bovenstaande stuk verwijderd, maar de toelichting staat nog in het voorbeeld.